metadeploy/api/serializers.py

```diff
@@ -502,16 +512,19 @@ def validate(self, data):
                 )
             )
 
-        user_has_valid_token = all(user.token)
-        if not user_has_valid_token:
-            raise serializers.ValidationError(
-                _("The connection to your org has been lost. Please log in again.")
-            )
+        # @@@ TODO: What to do with this in the case of a Scratch Org?
```

Comment:
This is making sure we still have oauth tokens for the org. In the case of a ScratchOrg they are in config["access_token"] and config["refresh_token"] as long as they haven't been removed yet.